controller.py

```diff
@@ -9,80 +9,107 @@ def __init__(self,path):
         self.path = path
         self.shortlist = load_shortlist(path)
         self.current_applicant = None
-        self.current_view = "home"
+        self.current_criterion = None
+        self.current_score = None
         self.view = View()
+        self.options = None # intial state
+        self.options_home = {"r":self.show_role_info,
+                            "a":self.show_applicants_list}
+        self.options_applist = {"b":self.show_boot_message,
```

Comment:
Here, and in variables and method names elsewhere, better to use the full word `applicant` because `app` is strongly associated with another meaning.